packages/gestalt/src/Masonry.js

```diff
@@ -176,7 +177,6 @@ export default class Masonry<T> extends React.Component<Props<T>, State<T>> {
   static defaultProps = {
     columnWidth: 236,
     // $FlowFixMe: new errors found from flow 0.96 upgrade
-    measurementStore: new MeasurementStore(),
```

Comment:
Since the prop `measurementStore` no longer has a default, flow type for the prop should be changed from required to optional

Reply:
Good call! Done.